### DISCUSSION TITLE
Language Insights within Org

### DISCUSSION THREAD

**Discussion by f-rich** [community/community#35762] [Other Feature Feedback, Questions, & Ideas]
I think it would be useful to have better insights into the languages/platforms etc. used within organisations.

Seeing the stats on languages used (rather than just the "Top Languages" summary, which has no numbers) would be a great start and potentially not too arduous (?) to implement.

Following on from that, seeing trends over time would allow orgs to see how their use of technology is evolving and perhaps identify areas for training etc.

I'm aware that there are some apps that have been written that do something similar, I'm suggesting this would be excellent to have it as native to the GitHub API/UI.

**Comment by jfullstackdev** [comment]
there is an external service for that but as far as I can remember, Orgs are not included,

I'm using this for my personal README file 

![image](https://user-images.githubusercontent.com/47092464/195132163-06f9e45f-3800-46fd-a145-fd8168beb4c6.png)

https://github.com/anuraghazra/github-readme-stats

**Reply by f-rich** [reply]
Thanks for responding.

Yeah, there's also https://github.com/manosprom/langstats that apparently works for an org.

I think there should be native ability within GitHub though, it'd be pretty useful.
